SmartEnrollmentFor911/SmartEnrollmentFor911/Pages/Index.cshtml.cs

```diff
@@ -13,24 +13,21 @@ namespace SmartEnrollmentFor911.Pages
 {
     public class IndexModel : PageModel
     {
-        private readonly ILogger<IndexModel> _logger;
+        //private readonly ILogger<IndexModel> _logger;
 
-        public IndexModel(ILogger<IndexModel> logger)
-        {
-            _logger = logger;
-        }
+        //public IndexModel(ILogger<IndexModel> logger)
+        //{
+        //    _logger = logger;
+        //}
```

Comment:
Delete this.  It's considered a no-no to commit commented code to version control.  You can always go back in history to see what used to be there.  Plus, you are in your own branch.